package.json

```diff
@@ -22,6 +22,7 @@
     "merge-stream": "0.1.6",
     "rimraf": "2.2.8",
     "run-sequence": "1.0.1",
+    "stylus": "~0",
```

Comment:
our CSS only works with certain stylus versions, so I wasn't expecting to override it.